src/status_im2/core.cljs

```diff
@@ -1,30 +1,33 @@
 (ns status-im2.core
   (:require
-    [utils.i18n :as i18n]
+    ;; NOTE: Do NOT sort i18n-resources because it MUST be loaded first.
```

Comment:
FYI reviewers @flexsurfer, @yqrashawn, @briansztamfater, @smohamedjavid, @J-Son89, @ibrkhalil, @ulisesmac, also @churik 

I just spent more than 2h to identify the culprit for the translations not being loaded in their entirety. The problem is that `i18n-resources` must be loaded before other namespaces in the entrypoint namespace `status-im2.core`. `i18n-resources` is quite the stateful namespace as it preloads a bunch of things at "load time", so it makes sense the problem was there.

So if any of you would like to automate cleaning namespaces on every file save using `clojure-lsp`, be aware `status-im2.core` should not be cleaned by it.

<img src="https://github.com/status-im/status-mobile/assets/46027/73c170b6-f79c-4d03-9651-3b676833e803" height="500" />